tesseract_ros/tesseract_examples/tesseract_ros_examples/src/pick_and_place_plan.cpp

```diff
@@ -270,7 +280,7 @@ int main(int argc, char** argv)
   }
 
   // Add a velocity cnt with time to insure that robot dynamics are obeyed
-  if (true)
+  if (false)
```

Comment:
Refering to line 288, we might make a note that you can set these in a yaml. I did this because I didn't know any better at the time. See [Trajopt 81](https://github.com/ros-industrial-consortium/trajopt_ros/issues/81)